### DISCUSSION TITLE
Vaadin Flow, Oauth2 con Spring Boot o Quarkus

### DISCUSSION THREAD

**Discussion by fredpena** [JavaDominicano/java-dominicano-comite-ejecutivo#33] [Ideas]
Durante la tercera semana de mayo en MonthlyTalks, se presentará sobre Vaadin Flow, Oauth2 con Spring Boot o Quarkus.

Favor agregar sus comentarios.

**Comment by ecabrerar** [comment]
Me gusta la propuesta, favor agregar una agenda de los temas a tratar.

**Comment by ecabrerar** [comment]
@fredpena tenemos este tema pendiente. Favor indicarnos la fecha en que te gustaría hacer la sesión.
